src/map-marker/styled-components.js.flow

```diff
@@ -5,11 +5,13 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
+import { number } from 'card-validator';
```

Comment:
```suggestion
```

Reply:
ah! this must have snuck in from vscode autocomplete